### PR TITLE
chore: Release of version 1.0.3

### PR DESCRIPTION
**Pull request by jancurn**
This PR contains:

- Bumped version to 1.0.3
- Updated CHANGELOG
- Checks for Node 16

I'm not sure what is the right release process now, please can you guys help?

Also, do you know how to change the build status badge in README to new GitHub build?

### CODE REVIEW

**Comment by szmarczak** [comment]
image

:D

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+1.0.3 / 2021-08-17
+==================
+- Fixed `EventEmitter` memory leak (see issue [#81](https://github.com/apify/proxy-chain/issues/81))
+- Added automated tests for Node 16
+- Updated dev dependencies
```
Comment (szmarczak) [inline CHANGELOG.md:5]: 🤔 Did you forget to push the commit?

**Comment by szmarczak** [comment]
> I'm not sure what is the right release process now

From [what I've seen in the workflow file](https://github.com/apify/proxy-chain/blob/00b5a13d50b08f2876516483d01ae6e25eb07cef/.github/workflows/release.yml#L103-L121) it automatically releases the beta when you bump the version. Then you draft a new release in the Releases tab (v1.0.3) and publish it - this will trigger a stable release.

**Comment by mnmkng** [comment]
https://www.notion.so/apify/Release-Process-7f2b48c4120b4ba6b9a7b3cae6c86583 it's the same as for all other libraries.

File: CHANGELOG.md (hunk 1 above)
Comment (jancurn) [reply CHANGELOG.md:5]: It was dependabot PRs merged directly to master before

**Review by szmarczak — APPROVED** [review]

**Comment by jancurn** [comment]
Guys please can you do the release? I have literally zero time for this today. Perhaps @szmarczak could become a maintainer of this library?

File: CHANGELOG.md (hunk 1 above)
Comment (B4nan) [reply CHANGELOG.md:5]: changelog is for end users, and they dont care about dev deps in general

(but let's keep it here, it won't hurt anyone)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+1.0.3 / 2021-08-17
+==================
+- Fixed `EventEmitter` memory leak (see issue [#81](https://github.com/apify/proxy-chain/issues/81))
```
Comment (B4nan) [inline CHANGELOG.md:3]: btw you can use just the `#81`, gh will do the link for you automatically if the issue/pr is in the same repository

**Review by B4nan — APPROVED** [review]

**Pull request merged**